src/lib.rs

```diff
@@ -234,7 +235,7 @@ impl<T: ?Sized> Bloom<T> {
 
     /// Clear all of the bits in the filter, removing all keys from the set
     pub fn clear(&mut self) {
-        self.bit_vec.clear()
+        self.bit_vec.fill(false)
```

Comment:
If you call `clear()` it works but... https://github.com/jedisct1/rust-bloom-filter/pull/34/commits/d58902a450a728e46997ee0a900796960477efab then you have to check whether it's defined or not, later.